cmd/restic/cmd_dump.go

```diff
@@ -54,12 +54,9 @@ func init() {
 
 func splitPath(p string) []string {
 	d, f := path.Split(p)
-	if d == "" {
+	if d == "" || d == "/" {
 		return []string{f}
 	}
-	if d == "/" {
-		return []string{d}
-	}
 	s := splitPath(path.Clean(d))
```

Comment:
There's some weird stuff going on when you pass odd paths, like `restic dump latest /foo/../`, which is caused by `path.Clean()` returning either an empty string or a relative path that makes no sense. You could do the following to always have it be absolute:
```suggestion
	s := splitPath(path.Clean(path.Join("/", d)))
```